jerry-core/parser/js/js-parser.c

```diff
@@ -2898,6 +2903,7 @@ parser_parse_script (const uint8_t *arg_list_p, /**< function argument list */
     {
       /* It is unlikely that memory can be allocated in an out-of-memory
        * situation. However, a simple value can still be thrown. */
+      JERRY_INIT_CURRENT_INSTANCE ();
```

Comment:
Same as befor the `JERRY_INIT_CURRENT_INSTANCE` is used twice. Just add the macro call at the start of the method.